addons/website_slides/views/website_slides_templates_lesson_fullscreen.xml

```diff
@@ -28,7 +28,7 @@
             <div class="o_wslides_fs_container d-flex position-relative overflow-hidden flex-grow-1">
                 <div class="o_wslides_fs_content align-items-stretch justify-content-center d-flex flex-grow-1 order-2"></div>
 
-                <div class="o_wslides_fs_sidebar text-white flex-shrink-0 order-1">
+                <div class="o_wslides_fs_sidebar o_wslides_fs_sidebar_hidden text-white flex-shrink-0 order-1">
```

Comment:
Just curious, why is not this implemented with a bootstrap tab behavior instead of custom JS ?

Reply:
The components has specific behaviours, overview: 

- sticky on **sm**, flex in **md** (resizing the page content).
- others elements act as toggles  (you can click on a custom backdrop to close the sidebar in mobile).
-  on **sm** you toggle `display`, but on **md** you toggle the sidebar's `width`...
- with this latest fix, the sidebar it's hidden by default in **sm** (but open in **md**)

We cold use "tab", but it really doesn't look and behave like a tab... 'collapse' it's probably a good candidate IMO, but It would require some `!important `overwrite too, and we'll end up writing some JS anyway ( to handle 'backdrop click', 'hidden by default' in **sm** etc..)